.github/workflows/ci.yml

```diff
@@ -62,12 +66,14 @@ jobs:
       HOME: "/root"
     steps:
       - uses: actions/checkout@v3
-        with: 
+        with:
           repository: waku-org/js-waku
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ env.NODE_JS }}
       - uses: ./.github/actions/npm
+      - name: Install headless-tests dependencies
```

Comment:
Since `headless-tests` is part of packages in root `package.json` - this step is not needed as `- uses: ./.github/actions/npm` should trigger installation